state/graphql-sender/src/test/java/com/expediagroup/streamplatform/streamregistry/state/graphql/ApolloExecutorTest.java

```diff
@@ -67,12 +71,12 @@ public void onResponseNoErrors() {
   public void onResponseErrors() {
     when(client.mutate(mutation)).thenReturn(mutationCall);
 
-    var result = underTest.execute(mutation);
+    CompletableFuture<? extends Response<?>> result = underTest.execute(mutation);
```

Comment:
Why not just use val like the others?

Reply:
The compiler couldn't determine the generic type. As with most of these cases, mash `?` `extends` `super` until it works 😆